service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -159,6 +315,24 @@ private ClaimEntity createClaim(Claim claim, VeteranEntity veteranEntity) {
     return claimRepository.save(claimEntity);
   }
 
+  private ExamOrderEntity createExamOrder(ExamOrder examOrder) {
+    // Currently ExamOrders only come from MAS
+    Optional<ClaimSubmissionEntity> claimSubmission =
+        claimSubmissionRepository.findFirstByReferenceIdAndIdTypeOrderByCreatedAtDesc(
+            examOrder.getCollectionId(), Claim.DEFAULT_ID_TYPE);
```

Comment:
If we assume this method is only called for v2 endpoints (which is a valid assumption since v1 doesn't involve exam ordering), then
```suggestion
            examOrder.getCollectionId(), Claim.V2_ID_TYPE);
```